src/archive.js

```diff
@@ -1,9 +1,16 @@
 /**
  * @fileoverview Keeps track of prior action states according to an
  * action's id
+ * @flow
  */
 
+import type Action from './action'
+
+type Payload = { [string]: * }
```

Comment:
Can you explain this bit to me? So you are defining a `Payload` type and it is an object that has...and then I don't know what the rest means.